### PR TITLE
DOCS: Proofreading Samples Python - 22.2

### PR DESCRIPTION
**Pull request by msmykx-intel**
Proofreading of Samples Python for 22.2

### CODE REVIEW

**Review by totoka-intel — APPROVED** [review]

File: samples/python/classification_sample_async/README.md [hunk 1]
```diff
@@ -9,24 +9,29 @@ The following Python API is used in the application:
 | :----------------- | :------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------ | :------------------------ |
 | Asynchronous Infer | [openvino.runtime.AsyncInferQueue], [openvino.runtime.AsyncInferQueue.set_callback], [openvino.runtime.AsyncInferQueue.start_async], [openvino.runtime.AsyncInferQueue.wait_all], [openvino.runtime.InferRequest.results] | Do asynchronous inference |
 
-Basic OpenVINO™ Runtime API is covered by [Hello Classification Python* Sample](../hello_classification/README.md).
+Basic OpenVINO™ Runtime API is descirbed in [Hello Classification Python Sample](../hello_classification/README.md).
```
Comment (helena-intel) [inline samples/python/classification_sample_async/README.md:12]:
```suggestion
Basic OpenVINO™ Runtime API is described in [Hello Classification Python Sample](../hello_classification/README.md).
```

File: samples/python/classification_sample_async/README.md [hunk 2]
```diff
@@ -78,12 +78,12 @@ To run the sample, you need specify a model and image:
    omz_downloader --name alexnet
    ```
 
-3. If a model is not in the IR or ONNX format, it must be converted. You can do this using the model converter:
+3. If a model is not in the OpenVINO IR or ONNX format, it must be converted with Model Converter:
```
Comment (helena-intel) [inline samples/python/classification_sample_async/README.md:81]: This changes the meaning. You definitely do not need Model Converter to convert a model to IR, this is just one way of doing this. I would recommend leaving this as is.

File: samples/python/classification_sample_async/README.md [hunk 3]
```diff
@@ -78,12 +78,12 @@ To run the sample, you need specify a model and image:
...
    ```
    omz_converter --name alexnet
    ```
 
-4. Perform inference of `banana.jpg` and `car.bmp` using the `alexnet` model on a `GPU`, for example:
+4. Perform inference of the `banana.jpg` and the `car.bmp`, using the `alexnet` model on a `GPU`, for example:
```
Comment (helena-intel) [inline samples/python/classification_sample_async/README.md:86]:
```suggestion
4. Perform inference on the `banana.jpg` and the `car.bmp` images, using the `alexnet` model on a `GPU` device, for example:
```

File: samples/python/classification_sample_async/README.md (hunk 2 above)
Comment (msmykx-intel) [reply samples/python/classification_sample_async/README.md]: Done in https://github.com/openvinotoolkit/openvino/pull/14168/commits/f15873dd7ee0277c2b5471c4367febc2e75ef7e4

**Pull request merged**